src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -775,4 +772,19 @@
     <value>Rename "{}"</value>
     <comment>Use '{}' to denote placement for the color scheme name to be renamed</comment>
   </data>
+  <data name="ColorScheme_DeleteButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Delete the current color scheme.</value>
+  </data>
+  <data name="ColorScheme_Rename.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Rename</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This profile cannot be deleted because it is shipped in-box.</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerDynamic" xml:space="preserve">
+    <value>This profile cannot be deleted because it is a dynamic profile.</value>
+  </data>
+  <data name="ColorScheme_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This color scheme cannot be deleted or renamed because it is shipped in-box.</value>

```

Comment:
once again "shipping" and "in-box" don't mean things to like 99% of our userbase

Reply:
Can you use the phrase "it's preinstalled with Windows Terminal" or can we simply hide the buttons on this schemes/profiles?

Reply:
"This profile cannot be deleted because it is included by default." ?